From 8dc21dd5a0e8be104571e62a55004b3b6fc9c5ef Mon Sep 17 00:00:00 2001
From: Mingcong Bai <jeffbai@aosc.io>
Date: Wed, 4 Sep 2024 11:57:12 +0800
Subject: [PATCH 13/13] fix(FindCrossGUID.cmake): do not append -dgb suffix

We build our packages with the RelWithDebInfo build type, which is kind of
not debug.
---
 cmake/modules/FindCrossGUID.cmake | 5 -----
 1 file changed, 5 deletions(-)

diff --git a/cmake/modules/FindCrossGUID.cmake b/cmake/modules/FindCrossGUID.cmake
index 38444f4433..9731d5d715 100644
--- a/cmake/modules/FindCrossGUID.cmake
+++ b/cmake/modules/FindCrossGUID.cmake
@@ -14,7 +14,6 @@ macro(buildCrossGUID)
   SETUP_BUILD_VARS()
 
   set(CROSSGUID_VERSION ${${MODULE}_VER})
-  set(CROSSGUID_DEBUG_POSTFIX "-dgb")
 
   set(_crossguid_definitions HAVE_NEW_CROSSGUID)
 
@@ -53,10 +52,6 @@ if(NOT TARGET CrossGUID::CrossGUID)
                                            HINTS ${DEPENDS_PATH}/lib ${PC_CROSSGUID_LIBDIR}
                                            ${${CORE_PLATFORM_LC}_SEARCH_CONFIG}
                                            NO_CACHE)
-    find_library(CROSSGUID_LIBRARY_DEBUG NAMES crossguidd crossguid-dgb
-                                         HINTS ${DEPENDS_PATH}/lib ${PC_CROSSGUID_LIBDIR}
-                                         ${${CORE_PLATFORM_LC}_SEARCH_CONFIG}
-                                         NO_CACHE)
 
     # NEW_CROSSGUID >= 0.2.0 release
     if(EXISTS "${CROSSGUID_INCLUDE_DIR}/crossguid/guid.hpp")
-- 
2.46.0

